cli/src/pcluster/api/controllers/image_operations_controller.py

```diff
@@ -230,18 +229,48 @@ def describe_image(image_id, region=None):
 
     :rtype: DescribeImageResponseContent
     """
+    imagebuilder = ImageBuilder(image_id=image_id)
+
+    try:
+        return _image_to_describe_image_response(imagebuilder)
+    except NonExistingImageError:
+        try:
+            return _stack_to_describe_image_response(imagebuilder)
+        except NonExistingStackError:
+            raise NotFoundException("No image or stack associated to parallelcluster image id {}.".format(image_id))
+
+
+def _image_to_describe_image_response(imagebuilder):
+    return DescribeImageResponseContent(
+        creation_time=imagebuilder.image.creation_date,
+        image_configuration=ImageConfigurationStructure(s3_url=imagebuilder.config_url),
+        image_id=imagebuilder.image_id,
+        image_build_status=ImageBuildStatus.BUILD_COMPLETE,
+        ec2_ami_info=Ec2AmiInfo(
+            ami_name=imagebuilder.image.name,
```

Comment:
minor - let's consider to add the "EC2 AMI description" as well

Reply:
Added